src/main/java/io/pravega/connectors/flink/FlinkPravegaReader.java

```diff
@@ -344,6 +345,8 @@ public void cancel() {
 
     @Override
     public void open(Configuration parameters) throws Exception {
+        deserializationSchema.open(RuntimeContextInitializationContextAdapters.deserializationAdapter(
```

Comment:
Please add unit tests for this `open` method to ensure it is called.
You can take use of Flink's `org.apache.flink.streaming.util.MockDeserializationSchema` to test.
Same for writer.